openedx_learning/apps/authoring/backup_restore/toml.py

```diff
@@ -64,28 +67,30 @@ def _get_toml_publishable_entity_table(
     entity_table.add("can_stand_alone", entity.can_stand_alone)
     # Add key since the toml filename doesn't show the real key
     entity_table.add("key", entity.key)
+    entity_table.add("created", entity.created)
 
     if not include_versions:
         return entity_table
 
-    current_draft_version = publishing_api.get_draft_version(entity)
-    current_published_version = publishing_api.get_published_version(entity)
-
-    if current_draft_version:
+    if draft_version:
         draft_table = tomlkit.table()
-        draft_table.add("version_num", current_draft_version.version_num)
+        draft_table.add("version_num", draft_version.version_num)
         entity_table.add("draft", draft_table)
 
     published_table = tomlkit.table()
-    if current_published_version:
-        published_table.add("version_num", current_published_version.version_num)
+    if published_version:
+        published_table.add("version_num", published_version.version_num)
     else:
         published_table.add(tomlkit.comment("unpublished: no published_version_num"))
     entity_table.add("published", published_table)
     return entity_table
 
 
-def toml_publishable_entity(entity: PublishableEntity, versions_to_write: list[PublishableEntityVersion]) -> str:
+def toml_publishable_entity(
+        entity: PublishableEntity,
+        versions_to_write: list[PublishableEntityVersion],
+        draft_version: Optional[PublishableEntityVersion],
+        published_version: Optional[PublishableEntityVersion]) -> str:
```

Comment:
I'll merge this anyway, but a nit here to prefer the newer notation of `PublishableEntityVersion | None` for annotations generally.